traptor/traptor.py

```diff
@@ -1393,7 +1388,7 @@ def createArgumentParser():
     parser = argparse.ArgumentParser()
     parser.add_argument(
             '--skipdelay',
-            action='store_true',  # which defaults to False.
+            action='store_false',  # which defaults to True.
```

Comment:
If I'm reading this properly, we now have a CLI argument `--skipdelay` which *adds* a delay to the startup; if that's correct I think it probably needs to be renamed, eg to `--delay` or something along those lines?

Reply:
Yeah, but I wanted to minimize my footprint.

Reply:
I think it needs to go away entirely. I will rename the flag.